odoo/_monkeypatches/lxml.py

```diff
@@ -0,0 +1,35 @@
+import inspect
+import lxml.html.clean
+import re
+
+from importlib.metadata import PackageNotFoundError, version
+
+from odoo.tools import parse_version
+
+# the "clean" module overrides its own import with the "Cleaner" class
+# fetch the module via object inspection...
+lxml_clean_module = inspect.getmodule(lxml.html.clean.clean)
+
+
+def patch_lxml():
+    # between lxml [4.6.0, 5.3.0[ having a couple data urls in a style attribute
+    # or style node removes the attribute or node erroneously
+    if parse_version(version('lxml')) >= parse_version("4.6.5"):
+        lxml_clean_broken = True
+        try:
+            # this is what implements the cleaner from lxml 5.2.0 onwards
+            if parse_version(version('lxml_html_clean')) >= parse_version("0.1.1"):
+                lxml_clean_broken = False
+        except PackageNotFoundError:
+            pass
+    if not lxml_clean_broken:
+        return
+
+    # double check that we're patching what we expect, we don't want to override it if it changed
+    if (
+        hasattr(lxml_clean_module, '_find_image_dataurls')
+        and hasattr(lxml_clean_module._find_image_dataurls, '__self__')
+        and type(lxml_clean_module._find_image_dataurls.__self__) is re.Pattern
+        and lxml_clean_module._find_image_dataurls.__self__.pattern == 'data:image/(.+);base64,'
+    ):
```

Comment:
Can you just remove this condition? Since this is expected to be a function, we can always replace it.